doc/onboarding_guide.md

```diff
@@ -1,7 +1,9 @@
 Onboarding Best Practices
 =========================
 
-Reach out to `azpycli@microsoft.com` to get the process started. You'll be assigned a dev contact on the CLI team. Early and frequent communication with this contact is essential to ensuring a smooth onboarding.
+As a prerequisite, please contact Carl Shipley first for the [PLR](http://aka.ms/plrcriteria) (Product Launch Readiness) process.
+
+Then reach out to `azpycli@microsoft.com` to get the CLI onboarding process started. You'll be assigned a dev contact on the CLI team. Early and frequent communication with this contact is essential to ensuring a smooth onboarding.
```

Comment:
ensuring -> ensure?

Reply:
It looks like both usage are OK. In the case of `ensuring`, the `ensuring a smooth onboarding` part is called a `gerund phrase`, which functions as nouns.

Reply:
Professional!